lib/conjur/cli.rb

```diff
@@ -155,7 +155,11 @@ def command_version_compatible? command
         run_default_handler = false
       elsif exception.is_a?(RestClient::Exception) && exception.response
         err = Conjur::Error.create exception.response.body
-        if err
+
+        if exception.http_code == 401
```

Comment:
I think we still want to make sure the exception gets passed along to GLI, to allow use of `GLI_DEBUG` to show the stack trace. Rescuing `RestClient::Unauthorized` in the `around` block above would allow us to do that.